lib/provisioner/worker/plugins/automators/chef_solo_automator/resources/cookbooks/coopr_hosts/recipes/default.rb

```diff
@@ -16,23 +16,19 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
-
+START = 60
 node['coopr']['cluster']['nodes'].each do |n, v|
   short_host = v.hostname.split('.').first
-  next unless v.key?('ipaddresses') && v['ipaddresses'].key?('access_v4')
-  hostsfile_entry v['ipaddresses']['access_v4'] do
-    hostname v.hostname
-    aliases [ short_host ]
-    unique true
-    priority 62
-    action :create
-  end
-  next unless v.key?('ipaddresses') && v['ipaddresses'].key?('bind_v4')
-  next if v['ipaddresses']['bind_v4'] == v['ipaddresses']['access_v4']
```

Comment:
Yeah. I couldn't think of a way to do a `next` in a sane manner here. It would end up being ugly...
psuedo:
```
if arr.length > 1
  some comparison between indexes
else
  just do it
end
```

It just seems like a bit too much when `hostsfile` is supposed to do this. We're definitely relying on the upstream code to Do The Right Thing™ here.